source/modules/MEI.header.xml

```diff
@@ -107,6 +107,62 @@
       </attDef>
     </attList>
   </classSpec>
+  <classSpec ident="att.perfRes" module="MEI.header" type="atts">
+    <desc xml:lang="en">Attributes that define the characteristics and components of the performance resource.</desc>
+    <classes>
+      <memberOf key="att.perfRes.basic"/>
+      <memberOf key="att.transposition"/>
+    </classes>
+    <attList>
+      <attDef ident="solo" usage="opt">
+        <desc xml:lang="en">Use this attribute to identify the performance resource as a soloist especially in an accompanied work, such as a concerto or vocal solo.</desc>
+        <datatype>
+          <rng:ref name="data.BOOLEAN"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.perfRes.basic" module="MEI.header" type="atts">
+    <desc xml:lang="en">Attributes that define the characteristics and components of the performance resource or a performance resource list.</desc>
+    <classes>
+      <memberOf key="att.adlibitum"/>
+    </classes>
+    <attList>
+      <attDef ident="count" usage="opt">
+        <desc xml:lang="en">Indicates the number of performers.</desc>
+        <datatype>
+          <rng:data type="positiveInteger"/>
+        </datatype>
+      </attDef>
+    </attList>
+  </classSpec>
+  <classSpec ident="att.adlibitum" module="MEI.header" type="atts">
+    <desc xml:lang="en">Attributes that describe a performance resource as ad libitum (optional).</desc>
+    <attList>
+      <attDef ident="adlib" usage="opt">
+        <desc xml:lang="en">Marks a performance resource as ad libitum (optional).</desc>
+        <datatype>
+          <rng:ref name="data.BOOLEAN"/>
+        </datatype>
+      </attDef>
+    </attList>
+    <remarks xml:lang="en">
+      <p>The technical term “ad libitum” has several meanings depending on the context in which it occurs. Currently only the use within a performance resource is supported.</p>
```

Comment:
The list below doesn't mention "performance resource" as one of the types, so I would either change this sentence to match the meaning from the list below, or change the list to mention a "performance resource." (I would personally go for the first, since I'm not sure it's very widely known what a "performance resource" is...)

Also, I would put this sentence after the list, not before it.

Reply:
the remarks have been added to declare that there are more dimensions in the meaning of `adlibitum`. After @kepper noticed that the content of the remarks lists only things that are not supported he asked me to clarify this. I think this will be lost if I skip the wording

Reply:
I will try to rephrase my question. 

You are saying in this sentence that the word "ad libitum" has several meanings, and that only one of them applies here, "use within a performance resource"

Then you list four meanings of ad libitum, but none of the four mention "use within a performance resource" directly. 

So I am unsure which of the four meanings you list actually apply. I think it's the first, but it would really help if there was a consistent way of talking about it between the sentence and the list.

Reply:
Wording will be revised, thanks for the thorough reading @ahankinson

Reply:
I got it. Done, thanks. Sorry for missunderstanding.